site/app/templates/UserProfile.twig

```diff
@@ -28,6 +28,12 @@
                             <i class="fas fa-pencil-alt"></i> Edit Preferred Names
                         </a>
                     </span>
+                    <span>&nbsp;&nbsp;&nbsp;</span>
+                    <span>
+                        <a class="icon" tabindex="0" href="javascript:showUpdateSecondaryEmailForm()" aria-label="Edit user secondary email" title="Edit user secondary email">
```

Comment:
This should probably be a button, `a` tags are reserved for navigation and buttons for everything else. Currently, you can not tab + open the modal with the keyboard which is an accessibility issue. Since the edit preferred names modal is setup the same way this can be done in a separate PR to fix both at the same time

Reply:
Alright. I'll create an issue for this and do that after this gets merged.